examples/on_gke_end_to_end/main.tf

```diff
@@ -132,6 +132,7 @@ module "gke" {
   remove_default_node_pool = true
   identity_namespace       = "${var.project_id}.svc.id.goog"
   node_metadata            = "GKE_METADATA_SERVER"
+  kubernetes_version       = "1.13"
```

Comment:
Does this need to be exactly 1.13, or can we do something like `~>1.13`?

Reply:
We found an issue with Kube 1.14 related to missing gCloud credentials.

Reply:
Ah interesting- thanks for catching that!

Reply:
For this, it doesn't accept the Terrafrom version syntax.  What this does is sets the major/minor.  But, it can be any patch release.

https://cloud.google.com/kubernetes-engine/versioning-and-upgrades